### ISSUE TITLE
Sanity-check configuration file

### ISSUE THREAD

**Issue by arter97** [cifsd-team/ksmbd-tools#248]
Hi, everyone.

I want to suggest a feature that's equivalent to Samba's testparm to sanity-check the configuration file.

While the documentation on Documentation/configuration.txt is quite comprehensive, it seems like ksmbd does not alert the users of any unrecognized values in the config file.

It would be a nice addition to ksmbd-tools (or ksmbd itself - e.g., warning in the kernel log) for new comers, migrating from other setups.

Thanks.

**Comment by namjaejeon** [comment]
Agreed, Good suggestion:), samba also ignores unsupported parameters without a message. but ksmbd is a latecomer. As you said, it seems to be needed. Let me check it.
